### DISCUSSION TITLE
Errors with wagmi's Polygon chain collection

### DISCUSSION THREAD

**Discussion by BLOCKCHAINDEVWEB** [wevm/wagmi#3029] [Q&A]
I have a difficult development with the wagmi chain collection.
I have several connectors and chains in my Wagmi setup, but I have the most errors with the Polygon network.

Has anyone encountered the same problems?

**Comment by snarflakes** [comment]
contractwrite isn't working for me with polygon, I see a pull request that renames the chain to Polygon POS but i can't tell if that has been pushed yet.
